pkg/api/frontendsettings.go

```diff
@@ -162,6 +162,7 @@ func (hs *HTTPServer) getFrontendSettingsMap(c *m.ReqContext) (map[string]interf
 			"hideFromList": panel.HideFromList,
 			"sort":         getPanelSort(panel.Id),
 			"dataFormats":  panel.DataFormats,
+			"state":        panel.State,
```

Comment:
@torkelo -- now most panel plugins get:
```
 'state': '',
```
is there a good way in go to only add stat if it exists?

Reply:
need to make dto for this and then we can add json attribute to not include the prop if empty. But leaving that as is for now.